docs/guide/search.md

```diff
@@ -223,12 +395,11 @@ Available tags for Sort Clause handlers in Legacy Storage Engine are:
 
 !!! note
 
-    You will find all the native handlers and the tags for the Legacy Storage Engine available in the eZ/Publish/Core/settings/storage\_engines/legacy/**search\_query\_handlers.yml** file.
+    You will find all the native handlers and the tags for the Legacy Storage Engine available in the `eZ/Publish/Core/settings/storage_engines/legacy/search_query_handlers.yml` file.
```

Comment:
It's a little bit outdated, since 2015 this file does not exist, instead we have:
- `criterion_handlers_common.yml`
- `criterion_handlers_content.yml`
- `criterion_handlers_location.yml`
- `sort_clause_handlers_common.yml`
- `sort_clause_handlers_content.yml`
- `sort_clause_handlers_location.yml`